leaf-ui/css/analysis_html.css

```diff
@@ -590,3 +590,28 @@ div.attribution{
   font-size: 9px;
   margin: 5px;
 }
+/* The left-side bar for analysisInspector and configInspector */
+#container-sidebar { 
+  left: auto; 
+  width: 200px; 
+  margin-top: 60px; 
+}
+
+#config-sidebar { 
+  margin-top:100px; 
+  width: 200px; 
+  position: absolute; 
+}
+
+#analysis-sidebar { 
+  margin-top: 0px; 
+}
+
+#analysis-sidebar a{
+  padding: 16px;
+  display: block;
+}
+
+#analysis-sidebar a:hover {
+  background-color: #E8E8E8;
+}
```

Comment:
I tried to delete this from both files (analysis_HTML.css and analysis_style.css) and it worked just fine.